R/tm_g_distribution.R

```diff
@@ -184,7 +203,10 @@ tm_g_distribution <- function(label = "Distribution Module",
   checkmate::assert_multi_class(pre_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
   checkmate::assert_multi_class(post_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
 
-  checkmate::assert_list(decorators, "teal_transform_module", null.ok = TRUE)
+  available_decorators <- c("histogram_plot", "qq_plot", "test_table", "summary_table")
+  decorators <- normalize_decorators(decorators)
+  assert_decorators(decorators, null.ok = TRUE, names = available_decorators)
```

Comment:
Should `null.ok = TRUE` by default for `assert_decorators`?

Reply:
It would make sense, I kept it as `FALSE`as that is the default for `checkmate::assert_*`

I'll leave it to you, thumbs up to change, down to keep it

Reply:
ooook, so this is with the convention to other `chekmate::asserts` . alrighty, let's have it `null.ok = FALSE` by default. Thanks for the reasoning.